app/components/header-nav.js

```diff
@@ -61,5 +79,29 @@ export default Component.extend({
         localStorage.setItem('locale', 'nl');
       }
     },
+    setAvailableStaticPages() {
+      if (!this.session.isAuthenticated && !this.media.isMobile) {
+        this.store.findAll('static-page').then((pages) => {
+          let newAvailableStaticPages = {};
+          // make key-value pairs for all found static pages
+          pages.forEach(
```

Comment:
Could be more compactly written as `let newPages = pages.reduce((obj, page) => Object.assign(obj, { [page.id]: page.title }), {})`